src/models/behaviors.js

```diff
@@ -13,6 +13,8 @@ var helpers = require('../util/helpers'),
     combinators = require('../util/combinators'),
     xpath = require('./xpath'),
     jsonpath = require('./jsonpath'),
+    csvToObject = require('csv-to-object'),
+    hashmap = require('hashmap'),
```

Comment:
When we made the change from a numbered index to key value pair we decided to use a hashmap instead.  We could still handle this as an object, but it would involve modifying the replace function to handle the differences between the way copyFn and lookupFn are handling keys.  If you would like that change we can modify some of the aforementioned functions.